### PR TITLE
Some fixes to robustify the addin

### PR DESCRIPTION
**Pull request by netonjm**
Includes:
Bump MonoDevelop.Addins nuget to 0.3.8
Fixes NRE parsing unwanted data from shared libraries	
Adds multiple Inline test cases to the test tree	
Fixed wrong test tree recovered from file
Fixes wrong testCaseMethodAttributeMarker			
Fixes TestMarkers to work correctly with Fact and Theory markers

### CODE REVIEW

**Comment by lextm** [comment]
Thanks. I would not be able to merge it till Xamarin/Mono fixes their build services.

**Comment by netonjm** [comment]
There is a way to chat with you?

**Comment by lextm** [comment]
@netonjm you can write mails to me support@lextm.com .

File: MonoDevelop.XUnit/XUnitTestLoader.cs [hunk 1]
```diff
@@ -98,31 +98,10 @@ void RunAsyncLoadTestInfo (XUnitTestInfoCache cache)
...
-							continue;
-						}
-					}
-
-#if EASY_DEBUGGING
 					var runner = new XUnitRunner.XUnitRunner();
 					testInfo = runner.GetTestInfo(testSuite.AssemblyPath, testSuite.SupportAssemblies.ToArray());
 					testSuite.OnTestSuiteLoaded(testInfo);
-#else
-					using (var runner = new ExternalTestRunner()) {
```
Comment (lextm) [inline MonoDevelop.XUnit/XUnitTestLoader.cs:120]: I don't think we should remove the external test runner part.

**Review by lextm — CHANGES_REQUESTED** [review]
Except one comment I left in https://github.com/xunit/xamarinstudio.xunit/pull/52/commits/e444e6cdd1828e993a35f14adb20140e37f7b67c I think all other changes look good.

If all code between #if EASY_DEBUGGING and #endif is kept, then I am happily accepting this PR.

**Review by lextm — APPROVED** [review]

**Comment by lextm** [comment]
Never mind. I will merge it and then revise the change.

**Pull request merged**